admin/includes/languages/english/lang.category_product_listing.php

```diff
@@ -10,6 +10,7 @@
     'HEADING_TITLE' => 'Categories / Products',
     'HEADING_TITLE_GOTO' => 'Go To:',
     'TABLE_HEADING_ID' => 'ID',
+    'TABLE_HEADING_IMAGE' => 'Image',
```

Comment:
Is it necessary to have the word "Image" as a column-heading?  ie: do people need to know that the pictures below that heading are images by giving the heading a title, instead of leaving it blank?

Reply:
(I'm thinking here about minimizing the addition of language strings that need translating/defining/upgrading.)

Reply:
I guess it's moot. If it's not there then someone's gonna want to add it for OCD reasons. :D

Reply:
On a tangent, I could prepare a PR that provides a link on the page to hide/show the images colum. I generally do not show images, for speed.